packages/qunit-dom/tsconfig.json

```diff
@@ -2,6 +2,16 @@
   "compilerOptions": {
     "noImplicitAny": true,
     "declaration": true,
+    // Forces import type when imports are only used as types.
+    "verbatimModuleSyntax": true,
+    // We build to ESM, so we need to tell TS that we build to ESM
+    // otherwise TS uses fake ESM (you use imports, but it assumes the compilation is to CJS)
+    // These are also required when using 
+    // verbatimModuleSyntax, because it's an 
+    // ESM-only feature
+    "target": "esnext",
```

Comment:
@NullVoxPopuli on line 19 there's `"target": "es5"` so this one is overridden

Reply:
thanks! https://github.com/mainmatter/qunit-dom/pull/2080

Reply:
Nice catch :)

Reply:
tbf it does stand out 😅 

```
❯ pnpm test

> qunit-dom@3.0.0 test (…)/qunit-dom/packages/qunit-dom
> vitest

▲ [WARNING] Duplicate key "target" in object literal [duplicate-object-key]

    tsconfig.json:19:4:
      19 │     "target": "es5"
         ╵     ~~~~~~~~

  The original key "target" is here:

    tsconfig.json:12:4:
      12 │     "target": "esnext",
         ╵     ~~~~~~~~
```